src/checks/kubernetes/services/ports/ports.go

```diff
@@ -38,12 +39,30 @@ type doesPortExistStruct struct {
 	} `yaml:"values"`
 }
 
+func doesPortExistParse(valuesYaml string, v *doesPortExistStruct) error {
+
+	err := yaml.Unmarshal([]byte(valuesYaml), &v)
+	if err != nil {
+		log.Fatalf("Unmarshal: %v", err)
+	}
+
+	if v.Values.ServiceName == "" {
+		return errors.New("Check values: invalid `ServiceName`")
+	}
+
+	if v.Values.Port == 0 {
```

Comment:
I'd go beyond this and check if its between `0 to 65353`

Reply:
Good idea